src/templates/post.tsx

```diff
@@ -192,7 +192,19 @@ const PageTemplate = ({ data, pageContext, location }: PageTemplateProps) => {
             <article css={[PostFull, !post.frontmatter.image && NoImage]}>
               <PostFullHeader className="post-full-header">
                 <PostFullTags className="post-full-tags">
-                  {post.frontmatter.tags && post.frontmatter.tags.length > 0 && (
+                  {post.frontmatter.tags && post.frontmatter.tags.length > 0 && config.showAllTags && (
+                    post.frontmatter.tags.map(
+                      function(t){
+                        return ([
+                          <Link to={`/tags/${_.kebabCase(t)}/`}>
```

Comment:
@MolarFox hey i'm going to merge this and change it but just as feedback you'd want to add a key https://reactjs.org/docs/lists-and-keys.html#keys and most would use a react fragment instead of an array

Reply:
Cheers for the feedback!

Reply:
thanks for the pr 👍